Assets/HoloToolkit/Input/Scripts/BasicCursor.cs

```diff
@@ -62,7 +62,7 @@ void LateUpdate()
             meshRenderer.enabled = GazeManager.Instance.Hit;
 
             // Place the cursor at the calculated position.
-            this.gameObject.transform.position = GazeManager.Instance.Position + GazeManager.Instance.Normal * DistanceFromCollision;
+            this.gameObject.transform.position = GazeManager.Instance.Position - Vector3.Normalize(GazeManager.Instance.Position) * DistanceFromCollision;
```

Comment:
This doesn't seem correct, converting the world space position to a normal like that will only work for a specific camera orientation.
You should probably compute a normal using something like Vector3.Normalize(Camera.main.transform.position - GazeManager.Instance.Position)